app/components/search.js

```diff
@@ -26,6 +26,12 @@ const Search = React.createClass({
     }
   },
 
+  handleKeyPress (event) {
+    if (event.keyCode === 13 && event.keyCode === 27) {
```

Comment:
Should this be a `||` not a `&&` ?

Also can you identify the 13 and 27? Magic numbers are scary :D

~~~ javascript
    const enterKey = 13
    const escKey = 27
    if (event.keyCode === enterKey || event.keyCode === escKey) {
      return false
    }
~~~